android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -284,7 +295,7 @@ public String getInstallReferrer() {
     constants.put("apiLevel", Build.VERSION.SDK_INT);
     constants.put("deviceLocale", this.getCurrentLanguage());
     constants.put("deviceCountry", this.getCurrentCountry());
-    constants.put("uniqueId", Secure.getString(this.reactContext.getContentResolver(), Secure.ANDROID_ID));
+    constants.put("uniqueId", Settings.Secure.getString(this.reactContext.getContentResolver(), Settings.Secure.ANDROID_ID));
```

Comment:
Can you revert this change ? If you feel that it's needed, please open a new PR for it and explain your motivations